tensorflow/lite/interpreter.h

```diff
@@ -335,6 +335,14 @@ class Interpreter {
   void UseNNAPI(bool enable);
 
   /// Set the number of threads available to the interpreter.
+  ///
+  /// NOTE: num_threads should be >= -1.
+  /// User may use -1, to reset to default values like below.
```

Comment:
Sorry that I misunderstood what you meant when writing these comments in your earlier comment. I thought you would use such structure in coding.  Here, having such pseudo-code-like comments looks like showing how to use this SetNumThreads API. So, what about simply saying "User may pass -1 to let the TFLite interpreter set the #threads available to itself."?